specs/challenge.md

```diff
@@ -45,39 +40,36 @@ The Colosseum contract implements the following interface:
 ```solidity
 interface Colosseum {
   event ChallengeCreated(
-    uint256 indexed outputIndex,
-    address indexed asserter,
+    uint256 indexed challengeId,
     address indexed challenger,
+    uint256 indexed outputIndex,
     uint256 timestamp
   );
 
-  event Bisected(uint256 indexed outputIndex, uint256 turn, uint256 timestamp);
-  event Proven(uint256 indexed outputIndex, bytes32 newOutputRoot);
-  event Approved(uint256 indexed outputIndex, uint256 timestamp);
-  event Deleted(uint256 indexed outputIndex, uint256 timestamp);
+  event Bisected(uint256 indexed challengeId, uint256 turn, uint256 timestamp);
+  event ProofCompleted(uint256 indexed challengeId, uint256 outputIndex);
+  event Closed(uint256 indexed challengeId, uint256 turn, uint256 timestamp);
 
   function createChallenge(
     uint256 _outputIndex,
     bytes32[] calldata _segments
-  ) external;
+  ) external payable;
 
-  function bisect(uint256 _outputIndex, uint256 _pos, bytes32[] calldata _segments) external;
+  function bisect(uint256 _pos, bytes32[] calldata _segments) external payable;
 
   function proveFault(
-    uint256 _outputIndex,
-    bytes32 _outputRoot,
     uint256 _pos,
     Types.OutputRootProof calldata _srcOutputRootProof,
     Types.OutputRootProof calldata _dstOutputRootProof,
     Types.PublicInput calldata _publicInput,
     Types.BlockHeaderRLP calldata _rlps,
     uint256[] calldata _proof,
     uint256[] calldata _pair
-  ) external;
+  ) external payable;
 
-  function challengerTimeout(uint256 _outputIndex) external;
+  function asserterTimeout() external;
 
-  function approveChallenge(uint256 _outputIndex) external;
+  function challengerTimeout(uint256 _challengeId) external;
```

Comment:
The Colosseum contract interface was updated in a recent commit.

Reply:
I almost made a big mistake. Thank you.
returned it back